storage/ndb/rest-server/rest-api-server/internal/handlers/handler.go

```diff
@@ -17,16 +17,29 @@
 
 package handlers
 
-import "net/http"
+import (
+	"errors"
+	"net/http"
+	"runtime/debug"
+
+	"hopsworks.ai/rdrs/internal/log"
+)
 
 type Handler interface {
 	Validate(request interface{}) error
 	Authenticate(apiKey *string, request interface{}) error
 	Execute(request interface{}, response interface{}) (int, error)
 }
 
-func Handle(h Handler, apiKey *string, request interface{}, response interface{}) (int, error) {
-
+func Handle(h Handler, apiKey *string, request interface{}, response interface{}) (statusCode int, err error) {
+	defer func() {
+		if r := recover(); r != nil {
+			statusCode = http.StatusInternalServerError
+			err = errors.New(`{"message": "Internal server error."}`)
```

Comment:
This is our current error return format?

Reply:
This is up for discussion.